src/operator/src/insert.rs

```diff
@@ -546,12 +543,7 @@ impl Inserter {
             | AutoCreateTableType::LastNonNull => {
                 for req in create_tables {
                     let table = self
-                        .create_non_logical_table(
-                            req,
-                            ctx,
-                            statement_executor,
-                            auto_create_table_type.clone(),
-                        )
+                        .create_physical_table(req, ctx, statement_executor)
```

Comment:
`req` made me think about the original request but not create table express.
```suggestion
                        .create_physical_table(create_table, ctx, statement_executor)
```